build_test/lib/src/resolve_source.dart

```diff
@@ -208,11 +215,14 @@ class _ResolveSourceBuilder<T> implements Builder {
 
   final onDone = new Completer<T>();
 
+  var hasStartedAction = false;
+
   _ResolveSourceBuilder(this._action, this._resolverFor, this._tearDown);
 
   @override
   Future<Null> build(BuildStep buildStep) async {
-    if (onDone.isCompleted) return;
+    if (hasStartedAction) return;
```

Comment:
Not sure this is really the right solution.... its arbitrary (although probably deterministic?) which input would actually get resolved.

I think we should instead force the user to provide a single primary input if they supply more than one input?

Reply:
Ah, we already had `_resolveFor`. Checking against that now.